From c07c5fd72fb5864005508dc4a792c11cfda92cb7 Mon Sep 17 00:00:00 2001
From: Pantelis Antoniou <panto@antoniou-consulting.com>
Date: Wed, 12 Jun 2013 11:12:26 +0300
Subject: [PATCH 09/24] capes: boneblack HDMI capes have blacklisted modes.

Added a few modes that are known not to work on my end.

Signed-off-by: Pantelis Antoniou <panto@antoniou-consulting.com>
---
 firmware/capes/cape-boneblack-hdmi-00A0.dts  | 11 +++++++++++
 firmware/capes/cape-boneblack-hdmin-00A0.dts | 11 +++++++++++
 2 files changed, 22 insertions(+)

diff --git a/firmware/capes/cape-boneblack-hdmi-00A0.dts b/firmware/capes/cape-boneblack-hdmi-00A0.dts
index 0cc9227..ce60528 100644
--- a/firmware/capes/cape-boneblack-hdmi-00A0.dts
+++ b/firmware/capes/cape-boneblack-hdmi-00A0.dts
@@ -111,6 +111,17 @@
 				pinctrl-0 = <&nxp_hdmi_bonelt_pins>;
 				pinctrl-1 = <&nxp_hdmi_bonelt_off_pins>;
 
+				modes-blacklisted =
+					"1920x1080@25",
+					"832x624@75";
+
+				// Enable this when we figure out the modes
+				//
+				// modes-whitelisted =
+				//	"1920x1080@24",
+				//	"1280x720@50", "1280x720@60",
+				//	"720x576@50","720x480@60";
+
 				panel-info {
 					bpp = <16>;
 					ac-bias = <255>;
diff --git a/firmware/capes/cape-boneblack-hdmin-00A0.dts b/firmware/capes/cape-boneblack-hdmin-00A0.dts
index 1feb013..b4305b0 100644
--- a/firmware/capes/cape-boneblack-hdmin-00A0.dts
+++ b/firmware/capes/cape-boneblack-hdmin-00A0.dts
@@ -96,6 +96,17 @@
 				pinctrl-0 = <&nxp_hdmi_bonelt_pins>;
 				pinctrl-1 = <&nxp_hdmi_bonelt_off_pins>;
 
+				modes-blacklisted =
+					"1920x1080@25",
+					"832x624@75";
+
+				// Enable this when we figure out the modes
+				//
+				// modes-whitelisted =
+				//	"1920x1080@24",
+				//	"1280x720@50", "1280x720@60",
+				//	"720x576@50","720x480@60";
+
 				panel-info {
 					bpp = <16>;
 					ac-bias = <255>;
-- 
1.9.0

